readme.md

```diff
@@ -121,6 +121,17 @@
 | ------: | -------: | ------: |
 |   3 - 6 |       10 | 60-120m |
 
+### [One Night Werewolf](https://beziergames.com/products/one-night-ultimate-werewolf)
+
+> One Night Ultimate Werewolf is a fast-paced game where everyone gets to be a different role. In the course of only one night and the following morning, the players will determine who among them is a werewolf...hopefully.
+
+![werewolf game image](https://cdn.shopify.com/s/files/1/0740/4855/products/ONUW_3D_Box_2048x.png?v=1461752063)
```

Comment:
The image here is huge, looks to be close to 1200px wide. Based on the link, it seems we could change this to have one that would be ~250px, as the formatting guide suggests.

```suggestion
![werewolf game image](https://cdn.shopify.com/s/files/1/0740/4855/products/ONUW_3D_Box_250x.png?v=1461752063)
```

Reply:
Thanks for your help! Had a lot of fun contributing :)